SAMtemplates/functions/main.yaml

```diff
@@ -99,10 +99,10 @@ Resources:
         Minify: true
         Target: "es2020"
         Sourcemap: true
-        tsconfig: clinicalViewLambda/tsconfig.json
+        tsconfig: clinicalView/tsconfig.json
         packages: bundle
         EntryPoints:
-          - clinicalViewLambda/src/handler.ts
+          - clinicalView/src/handler.ts
```

Comment:
`clinicalView` should be in the codeURI, this is currently deploying everything inside the packages folder to this lambda